packages/resolver/fs.js

```diff
@@ -35,15 +35,17 @@ class FS {
       file => file.match(".json") != null
     );
 
-    filteredBuildArtifacts.forEach(buildArtifact => {
+    for (let i = 0; i < filteredBuildArtifacts.length; i++) {
+      const filename = filteredBuildArtifacts[i];
+
       const artifact = JSON.parse(
-        fs.readFileSync(path.resolve(searchPath, buildArtifact))
+        fs.readFileSync(path.resolve(searchPath, filename))
```

Comment:
how is this any different?

Reply:
not sure, tbh. when rolling back the commits, this seemed to be the culprit. 🤷‍♂